atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -253,7 +252,18 @@ public void compactInternally() {
                         + " IOT tables to compensate for bloat. You can contact Palantir Support if you'd"
                         + " like more information. Underlying error was: {}", tableRef, e.getMessage());
             } catch (TableMappingNotFoundException e) {
-                throw Throwables.propagate(e);
+                throw new RuntimeException(e);
+            }
+        } else {
+            try {
+                log.warn("Tried to clean up table {}, but couldn't, because enterprise features are disabled. "
```

Comment:
Note that we shouldnt break the existing message here as deployments might be parsing logs for this.